code/modules/mob/living/carbon/human/human.dm

```diff
@@ -222,7 +222,7 @@
 
 	var/brute_loss = 0
 	var/burn_loss = 0
-	var/bomb_armor = getarmor(null, BOMB)
+	var/bomb_armor = 100 - (100/(1+(getarmor(null, BOMB)/50)))
```

Comment:
Gonna change this calculation to a define (and add a couple more defines for people to use) and apply it to a couple more cases I found after work.